### ISSUE TITLE
[v2.0] channel.html once again

### ISSUE THREAD

**Issue by uchigawa** [calzoneman/sync#176]
I think I wasn't precise enough when it comes to my last request about additional span12 div.

What I actually want is all the `main` and `playlistrow` content SEPARATELY in additional `span12` divs, not together in one `main-outer` div.

I prepared ready to use code:
http://pastebin.com/HBkSZTM5

Check if everything is logical and add it to the next commit. I added some `id`s to a few elements which do not have it. I added also some missing indent spaces for better code readability, so delete it if you consider them unnecessary.

**Comment by calzoneman** [comment]
Wait, why do the rows need to be in additional span12s?  Do border properties not work on the `div[class=row]` elements?

**Comment by calzoneman** [comment]
I'm confused as to the functional difference between

``` html
<div class="row" id="foo">
  <div class="span7">Something</div>
  <div class="span5">Something else</div>
</div>
```

and

``` html
<div class="row" id="baz">
  <div class="span12" id="bar">
   <div class="row" id="foo">
      <div class="span7">Something</div>
      <div class="span5">Something else</div>
    </div>
  </div>
</div>
```

**Comment by uchigawa** [comment]
In the first case all spans have layout-responding 30px(?) default left margin. If you set the margin manually, it can destroy all the layout. Not to mention about things like adding the simplest 1px border - a result is span5 BELOW span7.

In the second case only span12 has layout-responding left margin. Nested span7 in the `class="row"` has always 0px left margin. Whatever you do with CSS, it will not destroy layout.

Look here
http://twitter.github.io/bootstrap/scaffolding.html
in the Nesting columns section

I tested those 2 options and it behaved just like I wrote.

Edit: try to add in CSS editor
`#main {border:solid 1px red}`
and see what happens
